client/src/raw.rs

```diff
@@ -52,110 +50,110 @@ pub(super) mod sealed {
+    T::Error: Into<tonic::codegen::StdError>,
+    <T::ResponseBody as tonic::codegen::Body>::Error: Into<tonic::codegen::StdError> + Send,
+{
+    type SvcType = T;
 
-        fn operator_client(&mut self) -> &mut OperatorServiceClient<Self::SvcType> {
-            self.operator_svc_mut()
-        }
+    fn client(&mut self) -> &mut WorkflowServiceClient<Self::SvcType> {
+        self.workflow_svc_mut()
+    }
 
-        fn test_client(&mut self) -> &mut TestServiceClient<Self::SvcType> {
-            self.test_svc_mut()
-        }
+    fn operator_client(&mut self) -> &mut OperatorServiceClient<Self::SvcType> {
+        self.operator_svc_mut()
     }
 
-    impl<T> RawClientLike for ConfiguredClient<TemporalServiceClient<T>>
-    where
-        T: Send + Sync + Clone + 'static,
-        T: GrpcService<BoxBody> + Send + Clone + 'static,
-        T::ResponseBody: tonic::codegen::Body<Data = tonic::codegen::Bytes> + Send + 'static,
-        T::Error: Into<tonic::codegen::StdError>,
-        <T::ResponseBody as tonic::codegen::Body>::Error: Into<tonic::codegen::StdError> + Send,
-    {
-        type SvcType = T;
+    fn test_client(&mut self) -> &mut TestServiceClient<Self::SvcType> {
+        self.test_svc_mut()
+    }
+}
 
-        fn client(&mut self) -> &mut WorkflowServiceClient<Self::SvcType> {
-            self.client.client()
-        }
+impl<T> RawClientLike for ConfiguredClient<TemporalServiceClient<T>>
+where
+    T: Send + Sync + Clone + 'static,
+    T: GrpcService<BoxBody> + Send + Clone + 'static,
+    T::ResponseBody: tonic::codegen::Body<Data = tonic::codegen::Bytes> + Send + 'static,
+    T::Error: Into<tonic::codegen::StdError>,
+    <T::ResponseBody as tonic::codegen::Body>::Error: Into<tonic::codegen::StdError> + Send,
+{
+    type SvcType = T;
 
-        fn operator_client(&mut self) -> &mut OperatorServiceClient<Self::SvcType> {
-            self.client.operator_client()
-        }
+    fn client(&mut self) -> &mut WorkflowServiceClient<Self::SvcType> {
+        self.client.client()
+    }
 
-        fn test_client(&mut self) -> &mut TestServiceClient<Self::SvcType> {
-            self.client.test_client()
-        }
+    fn operator_client(&mut self) -> &mut OperatorServiceClient<Self::SvcType> {
+        self.client.operator_client()
     }
 
-    impl RawClientLike for Client {
-        type SvcType = InterceptedMetricsSvc;
+    fn test_client(&mut self) -> &mut TestServiceClient<Self::SvcType> {
+        self.client.test_client()
+    }
+}
 
-        fn client(&mut self) -> &mut WorkflowServiceClient<Self::SvcType> {
-            self.inner.client()
-        }
+impl RawClientLike for Client {
+    type SvcType = InterceptedMetricsSvc;
 
-        fn operator_client(&mut self) -> &mut OperatorServiceClient<Self::SvcType> {
-            self.inner.operator_client()
-        }
+    fn client(&mut self) -> &mut WorkflowServiceClient<Self::SvcType> {
```

Comment:
Might as well call this `workflow_client`, we ended up changing `service` to `workflow_service` in TS and Python when adding the other services.

Reply:
Done